cardshifter-server/src/main/java/com/cardshifter/server/model/MainServer.java

```diff
@@ -71,6 +71,11 @@ public MainServer(ServerConfiguration serverConfiguration) {
 	public Server start() {
         mods.loadExternal(mods.getDefaultModLocation());
         mods.loadExternal(Paths.get(config.getModsDirectory()));
+
+        if(mods.getAvailableMods().isEmpty()) {
+        	logger.warn("No mods were loaded from the different folders. Please check your installation.");
+        	throw new IllegalStateException("No mod loaded");
```

Comment:
I don't think `IllegalStateException` is correct here. It usually refers to the state of the application rather than the file system. Perhaps a `NoModsLoadedException` extending `RuntimeException` is better.

Nitpick: I would write "No mod**s** loaded".